indexer/services/comlink/src/controllers/api/v4/affiliates-controller.ts

```diff
@@ -116,38 +119,65 @@ class AffiliatesController extends Controller {
 
   @Get('/snapshot')
   async getSnapshot(
-    @Query() offset?: number,
+    @Query() addressFilter?: string[],
+      @Query() offset?: number,
       @Query() limit?: number,
-      @Query() sortByReferredFees?: boolean,
+      @Query() sortByAffiliateEarning?: boolean,
   ): Promise<AffiliateSnapshotResponse> {
-    const finalOffset = offset ?? 0;
-    const finalLimit = limit ?? 1000;
-    // eslint-disable-next-line
-    const finalSortByReferredFees = sortByReferredFees ?? false;
+    const finalAddressFilter: string[] = addressFilter ?? [];
+    const finalOffset: number = offset ?? 0;
+    const finalLimit: number = limit ?? 1000;
+    const finalsortByAffiliateEarning: boolean = sortByAffiliateEarning ?? false;
 
-    // simulate a delay
-    await new Promise((resolve) => setTimeout(resolve, 100));
+    // eslint-disable-next-line max-len
+    const infos: AffiliateInfoFromDatabase[] | undefined = await AffiliateInfoTable.paginatedFindWithAddressFilter(
```

Comment:
```suggestion
    const infos: AffiliateInfoFromDatabase[] | undefined = await AffiliateInfoTable
      .paginatedFindWithAddressFilter(
```
then you can delete the eslint-disable-next-line max-len